src/tests/Interop/UnmanagedCallersOnly/UnmanagedCallersOnlyTest.cs

```diff
@@ -45,11 +46,6 @@ public static int Main(string[] args)
                 TestUnmanagedCallersOnlyValid_ThrowException();
                 TestUnmanagedCallersOnlyViaUnmanagedCalli_ThrowException();
             }
-
-            if (args.Length != 0 && args[0].Equals("calli"))
```

Comment:
Was this being tested before?  (if not, you'll get a +1 in your counts)

Reply:
No, this was only for manual testing. In the interop space, we do very little automated testing of "kills the process" behaviors around invalid code patterns.

Reply:
The test count is a complete mess, under the merged scheme the test count is more than twice the original number. I think this is because multiple tests under this subtree used the ```[Fact]``` annotations even before test merging and in the legacy system those reported as a single test within a particular project whereas today they smoothly blend with the overall infra and report as multiple tests. There were two tests exercising CoreCLR runtime crashes under special circumstances, the bit you have pointed out and a similar bit in the SuppressGCTransition test. Apparently they haven't been running recently - we may have broken them at some point in the past by removing the RunOnly projects (tests running other tests with specific sets of command-line parameters) or perhaps they were always broken. So now they are broken no more and I expect follow-up work once they start throwing different errors on other platforms.

Reply:
OK, thanks @jkoritzinsky, so please let me know whether I should keep these two as negative tests or throw them away.

Reply:
I think it's worth keeping them as Pri1 tests, at least for now.